repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/hero-section.ftl

```diff
@@ -0,0 +1,14 @@
+<#macro heroSection heroSection>
```

Comment:
Will this ensure an empty hero section not be added in case if `Header` & `Text` fields of `HeroSection` are empty ?

Reply:
@smanivasag It is good to have that restriction. I have added that.